fastlane/lib/fastlane/actions/bundle_install.rb

```diff
@@ -28,7 +28,7 @@ def self.run(params)
 
           return sh(cmd.join(' '))
         else
-          Helper.log.info "No Gemfile found"
+          UI.message("No Gemfile found")
```

Comment:
Should this be an error or a warning ? I don't think it should be a standard message.